network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -125,6 +127,9 @@ class BlogPage(FoundationMetadataPageMixin, Page):
         help_text='Which blog topics is this blog page associated with? Please select 2 topics max.',
         blank=True,
         verbose_name='Topics',
+        # Limiting CMS choices to English, as topics get
+        # localized through the {% localized_version %} template tag.
+        limit_choices_to=models.Q(locale__id=default_locale.id),
```

Comment:
Hi Theo! I updated this to use the "default_locale" object which is English, similar to line 281 in this file.